GameFighter/Models/Army.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace GameFighter.Models
+{
+    public class Army
+    {
+        public List<Warrior> ArmyMembers { get; protected set; }
```

Comment:
Any class can reassign the collection, stored in this property. How can you prevent this?